packages/dd-trace/src/appsec/iast/analyzers/vulnerability-analyzer.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const Plugin = require('../../../../src/plugins/plugin')
+const { getFirstNonDDPathAndLine } = require('./../path-line')
+const { createVulnerability, addVulnerability } = require('../vulnerability-reporter')
+const overheadController = require('../overhead-controller')
+
+class Analyzer extends Plugin {
+  constructor (type) {
+    super()
+    this._type = type
+  }
+
+  _isVulnerable (value, context) {
+    return false
+  }
+
+  _report (value, context) {
+    const evidence = this._getEvidence(value)
+    const location = this._getLocation()
+    const vulnerability = createVulnerability(this._type, evidence, context.rootSpan.context().toSpanId(), location)
+    addVulnerability(context, vulnerability)
+  }
+
+  _getEvidence (value) {
+    return {
+      value,
+      ranges: null
```

Comment:
why return `ranges: null` and not ignore `ranges`? In this version we can ignore ranges, the current iast doesn't know about it yet